weave/trait/time/clock_format_trait.proto

```diff
@@ -0,0 +1,28 @@
+// This file specifies a Weave Common trait for setting the clock format.
```

Comment:
Please include a standard header per other traits in this repo.

Reply:
Done, not sure if there was a way to amend the first commit like in gerrit (force push?) so I just pushed another commit per the github instructions.